arangod/Cluster/ClusterMethods.cpp

```diff
@@ -3579,50 +3579,57 @@ arangodb::Result lockDBServerTransactions(std::string const& backupId,
   }
 
   // Perform the requests
-  cc->performRequests(requests, lockWait + 1.0, Logger::BACKUP, false, false);
+  cc->performRequests(requests, lockWait + 5.0, Logger::BACKUP, false, false);
 
-  // Now listen to the results:
+  // Now listen to the results and report the result:
+  arangodb::Result res(TRI_ERROR_NO_ERROR);
+  auto reportError = [&](std::string const& m) {
```

Comment:
Should this be a more general function like `arangodb::Result forceError(arangodbResult&& in, uintX_t error, std::string const& extra message)`

Reply:
No, this is just something I am using locally here. I doubt that it has wider sensible use.